src/utils/index.ts

```diff
@@ -139,3 +139,21 @@ export const getTotalSupply = async (
     return '0'
   }
 }
+
+export const getSupplyCap = async (
+  tokenAddress: string,
+  abi: AbiItem,
```

Comment:
I don't think you need to pass in the ABI here. Just import the ABI at the top of the file since the abi required to `getSupplyCap` shouldn't change.

Reply:
So then this should only accept token address + provider